api/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.gradle.kotlin.dsl.flyway
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.springframework.boot") version "2.7.0-SNAPSHOT"
+    id("org.springframework.boot") version "2.6.5-SNAPSHOT"
```

Comment:
Do you need `-SNAPSHOT`?

Reply:
Without `-SNAPSHOT`,SpringBoot cannnot be started